.travis.yml

```diff
@@ -16,3 +16,4 @@ script:
   - go generate -x ./... && git diff --exit-code; code=$?; git checkout -- .; (exit $code) # Check that go generate ./... produces a zero diff; clean up any changes afterwards.
   - go tool vet .
   - go test -v -race ./...
+  - go test -v -tags=integration -run=^$ ./test/integration # Check that integration test builds successfully, but don't run any of the tests (they hit live GitHub API).
```

Comment:
Have you seen my comment above at https://github.com/google/go-github/pull/674#issuecomment-316196325? Does it answer your question, or should I elaborate?

Note that `go test` is a superset of `go build`.